app/models/ingest_job.rb

```diff
@@ -985,10 +984,22 @@ def anndata_summary_props
     file_type = study_file.file_type
     trigger = study_file.upload_trigger
     job_status = previous_jobs.map(&:error).compact.any? ? 'failed' : 'success'
+    error_action = nil
```

Comment:
See comment beginning "AIUI".

```suggestion
    action = args.detect {|c| LifeSciencesApiClient::FILE_TYPES_BY_ACTION.keys.include?(c.to_sym) }
```